.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+**Replace this text with a summary of the changes in your PR.
+The more detailed you are, the better.**
+
+-----------------
+
+Before submitting the PR make sure the following are checked:
+
+* [ ] Wrote [good commit messages][1]. ([How to fix commits after they've been pushed](https://superuser.com/questions/751699/is-there-a-way-to-edit-a-commit-message-in-github/751909))
+* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
```

Comment:
Feel like `Fix` is a confusing - as some commit will be addressing enhancements

Reply:
I see your point, but I am not entirely convinced this will cause much confusion since, in order to make sense of the commit message, the exact issue will have to be visited, anyway.

Reply:
I think I've come around to your point; how do you feel about [issue #issue-number]? For example:

```
[issue #200] Add pull request template
```

I think, without context [#200] would be confusing (and thus people would omit it and we'd have to explain why we prefer it).